frontend/src/app/modules/home/home-page/home-page.component.ts

```diff
@@ -51,14 +51,21 @@ export class HomeComponent extends BaseComponent implements OnInit, OnDestroy {
                 this.organization = organization;
                 this.getAllDashboards();
             });
-        this.runBroadcastHub();
+        this.runHubs();
     }
 
-    runBroadcastHub() {
-        this.broadcastHub.start()
-            .then(() => this.broadcastHub.listenMessages(msg =>
-                this.toastNotificationService.info(`The next broadcast message was received: ${msg}`)))
-            .catch(() => this.toastNotificationService.error('BroadcastHub failed to start.'));
+    runHubs() {
+        this.runIssuesHub();
+    }
+
+    runIssuesHub() {
+        this.issuesHub.start()
+            .then(() => {
+                this.issuesHub.messages.pipe(this.untilThis).subscribe(issue => {
+                    this.toastNotificationService.info(`Received issue: ${issue.issueDetails.errorMessage}`);
+                });
+            })
+            .catch(() => this.toastNotificationService.error('Issues Hub failed to start.'));
```

Comment:
Please use async/await instead .then and .catch

Reply:
Done